### ISSUE TITLE
REQUEST - old windows installation screen

### ISSUE THREAD

**Issue by kendi182** [cschneegans/unattend-generator#116]
![387118445-3acc19c2-0b3a-4785-930b-a6a618733c08](https://github.com/user-attachments/assets/239e031b-99d1-48ae-ab6d-5a4f1a502664)

Is there a way to configure the xml to activate this option to use the old Windows installation and configuration screen?

ventoy 1.0.99 is having problems installing windows 24H2.

Until this bug is resolved, this alternative "solution" may help.

**Comment by cschneegans** [comment]
This has been [reported before](https://github.com/cschneegans/unattend-generator/issues/67).

I have not managed to switch from ConX to legacy setup mode using the autounattend.xml file, and I don't think this is even possible. If you want to use both an answer file and legacy setup mode with 24H2, I recommended the following steps:

1. Check the _Use filename notautounattend.xml rather than autounattend.xml_ setting in my generator. (Without this setting, the autounattend.xml file would immediately be processed by ConX setup, and you would have no chance to switch to legacy setup mode.)
2. When the ConX setup starts, click **Next** twice, then click **Previous Version of Setup**:
![image](https://github.com/user-attachments/assets/07396156-3309-47a6-b487-a791802914ad)
3. When legacy setup has launched, press Shift+F10 to open a command prompt, then type `setup.exe /Unattend:D:\notautounattend.xml` to continue with an answer file. (You may need to replace `D:\` with another drive.)

**Comment by kendi182** [comment]
```
<RunSynchronous>
				<RunSynchronousCommand wcm:action="add">
					<Order>1</Order>
					<Path>reg.exe add "HKLM\SYSTEM\Setup\LabConfig" /v BypassTPMCheck /t REG_DWORD /d 1 /f</Path>
				</RunSynchronousCommand>
				<RunSynchronousCommand wcm:action="add">
					<Order>2</Order>
					<Path>reg.exe add "HKLM\SYSTEM\Setup\LabConfig" /v BypassSecureBootCheck /t REG_DWORD /d 1 /f</Path>
				</RunSynchronousCommand>
				<RunSynchronousCommand wcm:action="add">
					<Order>3</Order>
					<Path>reg.exe add "HKLM\SYSTEM\Setup\LabConfig" /v BypassRAMCheck /t REG_DWORD /d 1 /f</Path>
				</RunSynchronousCommand>
				<RunSynchronousCommand wcm:action="add">
                    <Description>Switch to legacy Setup</Description>
                    <Order>4</Order>
                    <Path>reg.exe add "HKLM\SYSTEM\Setup" /v CmdLine /t REG_SZ /d "X:\sources\setup.exe" /f</Path>
                </RunSynchronousCommand>
			</RunSynchronous>
```

Have you tried using it this way?
I read on this forum that it seems to work for some users.

**Comment by kendi182** [comment]
The fix for this issue came in Ventoy version 1.1.04.
thanks!
